app/api/services/AuthService.scala

```diff
@@ -34,23 +34,31 @@ class AuthService @Inject() (val connector: AuthConnector, val appConfig: AppCon
     override def authConnector: AuthConnector = connector
   }
 
+  private val successOutcome = Future.successful(Right(UserDetails("Authorised", None)))
+
+  private lazy val callAuth = {
```

Comment:
Why do we need to lazily evaluate this?

Reply:
It's because the mocking (specifically calls to `mock[AuthService]`) will try to create an instance of `AuthService` but with `null` `appConfig`. So tests get a `NullPointerException` when they try to evaluate the `callAuth` val.

Reply:
Ah right yes makes sense now. Couldn't see that before!